resolwe/flow/filters.py

```diff
@@ -163,6 +163,22 @@ def filter_for_user(self, queryset, name, value):
         user = self.request.user
         return queryset.filter_for_user(user, Permission.from_name(value))
 
+    def filter_for_group(self, queryset, name, value):
+        """Filter queryset by group."""
+        return queryset.filter(permission_group__permissions__group=value)
+
+    def filter_private(self, queryset, name, value):
+        """Return only elements that are not public (explicitly shared with me)."""
+
+        public_user = get_user_model().objects.get(username="public")
+        public_filter = Q(permission_group__permissions__user=public_user)
+
+        # if filter value is False, return only public elements
```

Comment:
I would prefer the comment to be associated with the row immediately below the comment.
So maybe `If filter value is True, return only private elements` would be more appropriate.